src/NUnitFramework/testdata/SetUpFixtureData.cs

```diff
@@ -610,3 +610,154 @@ public void Test()
     }
 }
 #endregion
+
+#region SomeFixtureWithExceptions
+[TestFixture]
+public class SomeBaseFixtureExceptionInSetUp
+{
+	private readonly bool exceptionInFixtureSetUp;
```

Comment:
The coding standard is to also use an underscore for fields. This should be `_exceptionInFixtureSetUp`. Minor and not all code conforms, but we are trying to update as we go.